.gitignore

```diff
@@ -62,3 +62,7 @@ buck-out/
 # Expo
 .expo/*
 web-build/
+
+# npm/yarn locks
+package-lock.json
+yarn.lock
```

Comment:
would at least keep the yarn.lock, I believe it's important for making sure the developers have the correct package versions https://www.arahansen.com/the-ultimate-guide-to-yarn-lock-lockfiles/

Reply:
ah ok. I will include the lock.json files.